pom.xml

```diff
@@ -13,15 +13,15 @@
 
   <groupId>de.medizininformatik-initiative</groupId>
   <artifactId>flare</artifactId>
-  <version>2.4.0-alpha.2</version>
+  <version>2.4.0-alpha.3</version>
 
   <name>Flare</name>
   <description>Flare</description>
 
   <properties>
     <java.version>17</java.version>
     <testcontainers.version>1.20.3</testcontainers.version>
-    <ontology.version>3.0.0-alpha</ontology.version>
+    <ontology.version>v3.0.0-alpha.1</ontology.version>
```

Comment:
@alexanderkiel  needs to be 3.0.0-alpha.1

Reply:
fixed